ledger/src/tcpsigner/tcpsigner.c

```diff
@@ -60,6 +61,12 @@ typedef enum {
     ARG_NU_IRIS,
 } arg_non_printable_t;
 
+/**
+ * UI heartbeat memory area
+ * There's probably a better
```

Comment:
This comment looks incomplete, should it be "there's probably a better **place**"?

Reply:
Good catch, fixed!